tools/releaser/scripts/version-paths.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-export VERSION_UPDATE_PATHS="../**/*.go,../**/*.md,../tools/**/*.mustache,../go.mod,../.github/workflows/autoupdate-dev.yaml"
```

Comment:
.github/workflows/autoupdate-dev.yaml doesn't exist

Reply:
It was renamed to .github/workflows/autoupdate-preview.yaml

Reply:
ah, ok, included